install.sh

```diff
@@ -84,6 +85,7 @@ while true; do
     -d | --dependencies)             install_dependencies=true;                                          shift ;;
          --debug)                    build_release=false;                                                shift ;;
          --disable_backtrace)        build_bfd=false;                                                    shift ;;
+         --disable-colltrace)        collective_trace=false;                                             shift ;;
```

Comment:
Instead of adding a new commit addressing the reviewer's comments, it is possible to amend the existing commits. That way, when the PR is merged, you do not have to merge the intermediate versions of the commits, you will be merging the final version only. In my opinion, this way we will be merging a cleaner version of the PR and it is better in terms of long-term maintenance of the repo.

Reply:
@nusislam What about squash and merge? Wouldn't that also work?

Reply:
Probably not. If you have one commit addressing the reviews for three initial commits, squashing them will result into a single commit. But we need to keep the three initial commits. In open-source repositories like MPICH, they usually do interactive `rebase` to `amend` individual commits.

Reply:
I think if the PR is for one feature or issue and there isn't need to keep multiple commit in final version, then "squash and merge" should be fine. If in some cases we want to keep multiple commits for any reason, then we need to do rebase and force push